packages/styles/src/components/form-check.scss

```diff
@@ -152,6 +153,22 @@
       }
     }
 
+    @include utility-mx.high-contrast-mode {
+      transition: none;
+      border: form-check.$form-check-input-border-width solid CanvasText;
+
+      &:checked {
+        background-color: CanvasText;
```

Comment:
Maybe we can use AccentColor and as a fallback to CanvasText. I'm not sure of what I'm saying, but it looks like it's the color to show that an element is active. It's supported by Firefox (though I don't see the difference) and under a flag on blink engine browsers.